conf/docker/Dockerfile

```diff
@@ -43,7 +43,7 @@ ENV POETRY_VERSION=2.1.1 \
 # Install pipx and poetry
 RUN python3 -m pip install --user pipx && python3 -m pipx ensurepath
 ENV PATH="${PIPX_ROOT_PATH}:${VENV_PATH}/bin:$PATH"
-RUN pipx install poetry==2.1.1
+RUN pipx install poetry
```

Comment:
We need to keep the Poetry version pinned here, since pipx isn't going to recognize the Poetry-specific environment variable. I think we might not need to set `POETRY_VERSION` but I haven't confirmed.

Reply:
Ah good catch, this env var looks to be deprecated now anyway